wordpress-mobile/react-native-keyboard-aware-scroll-view
# Fix uimanager warning about calling measureSelectionInWindow

This PR fixes the following warning that started occurring after updating the main app to react native 0.49.

![Simulator Screen Shot - iPhone X - 2019-03-18 at 19 40 10](https://user-images.githubusercontent.com/5032900/54620258-f778db80-4a76-11e9-8e09-fa7a73f5b690.png)

As of react native 0.49, methods that are called via UIManager is restricted, currently this only causes a warning but in the next releases this will be forbidden completely. So we are adding our own native module and calling the measureSelectionInWindow via that module. This was actually the proper way to call it, the previous version was preferred only for being very practical. 

I've generated a new xcodeproj with react-native-create-library and updated the prefix as RNT. This wasn't too necessary I guess but it looked like a good idea to create the native module according to the docs.

**TO TEST**

- Tap some place on a paragraph block and observe keyboard has opened
- Close the keyboard and see that the warning is gone

Test with both:

gutenberg-mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/761
WPiOS PR: https://github.com/wordpress-mobile/WordPress-iOS/pull/11298
